product_docs/docs/pgd/5/security/role-management.mdx

```diff
@@ -2,36 +2,36 @@
 title: Role management
 ---
 
-Users are global objects in a PostgreSQL instance. A `CREATE ROLE` command or its alias `CREATE USER` is replicated automatically if they're executed in a PGD replicated database. If a role or user is created in a non-PGD, un-replicated database, the role will only exist for that PostgreSQL instance. `GRANT ROLE` and `DROP ROLE` work in the same way, only replicating if applied to a PGD replicated database.
+Users are global objects in a PostgreSQL instance. A `CREATE ROLE` command or its alias `CREATE USER` is replicated automatically if it's executed in a PGD replicated database. If a role or user is created in a non-PGD, unreplicated database, the role exists only for that PostgreSQL instance. `GRANT ROLE` and `DROP ROLE` work the same way, replicating only if applied to a PGD-replicated database.
 
 !!! Note
-Remember that a user in Postgres terms is simply a role with login privileges.
+A user in Postgres terms is a role with login privileges.
 !!!
 
-## Role Rule - No un-replicated roles
+## Role rule - No un-replicated roles
 
-It is especially important that if you do create a role or user in a non-PGD, un-replicated database, that you do not go on to make an object in the PGD replicated database rely on that role. It will break the replication process as PGD cannot replicate a role that is not in the PGD replicated database. 
+If you do create a role or user in a non-PGD, unreplicated database, it's especially important that you don't make an object in the PGD-replicated database rely on that role. It will break the replication process, as PGD can't replicate a role that isn't in the PGD-replicated database. 
 
-This automatic replication behavior can be disabled by turning off the[ bdr.role_replication](https://www.enterprisedb.com/docs/pgd/latest/reference/pgd-settings/#bdrrole_replication) setting but is not recommended.
+You can disable this automatic replication behavior by turning off the [`bdr.role_replication`](https://www.enterprisedb.com/docs/pgd/latest/reference/pgd-settings/#bdrrole_replication) setting. However, we don't recommend that.
```

Comment:
As above, but moreso that we want a direct association with the statement before. The however disassociates it in the mind of the reader.

Reply:
```suggestion
You can disable this automatic replication behavior by turning off the [`bdr.role_replication`](https://www.enterprisedb.com/docs/pgd/latest/reference/pgd-settings/#bdrrole_replication) setting, but we don't recommend that.
```